plone/app/locales/locales/nl/LC_MESSAGES/widgets.po

```diff
@@ -155,15 +159,15 @@ msgstr "Kies bestand"
 
 #: ./patterns/pickadate/pattern.js
 msgid "Clear"
-msgstr ""
+msgstr "Leegmaken"
 
 #: ./patterns/thememapper/js/cacheview.js
 msgid "Clear Cache"
-msgstr ""
+msgstr "Cache leegmaken"
 
 #: ./patterns/thememapper/pattern.js
 msgid "Clear cache"
-msgstr ""
+msgstr "Cache leegmaken"
```

Comment:
Dubbeling met string hierboven alleen hoofdletter.  Ik probeer terug te vinden waar deze vertalingen vandaan komen om engels te verbeteren, maar waar staat dit. widgets.po, mockup, plone.app.widgets, plone.staticresources?  Ik zou denken plone.app.widgets, maar daar staat geen js in.  :-S